torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -11,6 +13,22 @@
 __all__ = []
 
 
+class _Wav2Vec2Model(Module):
+    """Wrapper class for Wav2Vec2Model
+
+    This class allows for normalization at the input (see _params["normalize_waveform"])
+    """
+
+    def __init__(self, model: Wav2Vec2Model, normalize_waveform: Optional[bool]):
+        self.model = model
+        self.normalize_waveform = normalize_waveform
+
+    def forward(self, waveforms: Tensor) -> Tuple[Tensor, Optional[Tensor]]:
```

Comment:
lengths is also included in the Wav2Vec2Model's input arguments. could you add it?